### ISSUE TITLE
Buttons do not have an accessible name

### ISSUE THREAD

**Issue by CedrikNikita** [aeternity/superhero-wallet#2676]
A lot of buttons in our project doesn't have accessible name, and they don't follow certain rules to improve overall accessibility.
https://dequeuniversity.com/rules/axe/4.8/button-name
